### DISCUSSION TITLE
Review and Approval Request for Vite.js Project Template Alternatives

### DISCUSSION THREAD

**Discussion by Showkath** [UI5/webcomponents-react#5905] [Q&A]
Hello Experts,

We are considering using the Vite.js Project [template](https://sap.github.io/ui5-webcomponents-react/?path=/docs/project-templates-examples--docs) from SAP/ui5-webcomponents-react [[https://github.com/SAP/ui5-webcomponents-react/tree/main/templates/vite-ts](https://github.com/SAP/ui5-webcomponents-react/tree/main/templates/vite-ts)] . However, a colleague has suggested an alternative: the Vite template from RicardoValdovinos [[https://github.com/RicardoValdovinos/vite-react-boilerplate/tree/main](https://github.com/RicardoValdovinos/vite-react-boilerplate/tree/main)] . Could you please review this alternative template for any security compliance issues according to SAP Policy? Additionally, please provide approval if it meets our standards. My colleague believes the RicardoValdovinos template is more powerful, but I want to ensure it is suitable for SAP internal projects.

Thank you.

**Comment by Lukas742** [comment]
Hi @Showkath 

Our templates are provided as examples of how to use ui5-webcomponents-react (UI5WCR). They primarily include internal features such as theming, but some templates also implement specific framework functionalities like routing or e.g. utilize other frameworks for testing. There are numerous templates available that demonstrate more complex examples and incorporate useful third-party libraries. It's up to you to decide whether to start from scratch, use a public template without UI5WCR, or use one of our templates.

What we can’t offer is a compliance check, so if you’re using one of these templates, you have to make sure that you meet SAP compliance criteria yourself. You can find the guidelines by searching for 'Open Source Policy' in our internal portal.
